### PR TITLE
ASM-3701 VirtIscsiMacAdrr was sometimes being set as empty string

### PR DESCRIPTION
**Pull request by asaenz**

### CODE REVIEW

**Comment by gavin-scott** [comment]
@asaenz what's the jira number on this?

**Comment by asaenz** [comment]
@gavin-scott Added the jira number to the title

File: lib/puppet/provider/importtemplatexml.rb [hunk 1]
```diff
@@ -372,12 +370,21 @@ def munge_iscsi_partitions(nc, changes)
   end
 
   def munge_virt_mac_addr(nc, changes)
+    virtMacAddr = ''
+    virtIscsiMacAddr = ''
```
Comment (gavin-scott) [inline lib/puppet/provider/importtemplatexml.rb]: Can these be removed? Looks like they are unconditionally set below before they are used.

Comment (gavin-scott) [inline lib/puppet/provider/importtemplatexml.rb]: Sounds like we need this initialization, but should be moved into the partition loop below.

**Comment by asaenz** [comment]
@gavin-scott Made the change to move the initialization into the loop

**Pull request merged**